### DISCUSSION TITLE
Should I change db.json without starting the program?

### DISCUSSION THREAD

**Discussion by tentasys** [massCodeIO/massCode#482] [Q&A]
I made Google drive as my db.json stroage, and there is an issue on my google drive - it doesn't monunt on my local environment.
So when I start this program, I can see below message.

![image](https://github.com/user-attachments/assets/8e59deda-9882-4062-bdc2-25eb81ca9ef8)

I would like to change db.json path to other path.
I tried to reset db.json path to initial path, so I re-installed this program several times, but it didn't work. It still use the same db.json path.
How can I change db.json path without starting this program?

**Comment by Airr** [comment]
> How can I change db.json path without starting this program?

Hi, @tentasys 

You can try modifying the preferences.json file.  Locations for each supported OS are:

Linux:  $HOME/snap/masscode/x1/.config/massCode/v2/preferences.json
Windows: %AppData%\Masscode\v2\preferences.json
Mac: $HOME/Library/Application Support/massCode/v2/preferences.json

Make sure that massCode is not running, then:

Modify the "storagePath" key with the path you want to use.  If the directory doesn't exist, it will be created with the default db.json file.

**Comment by pa-0** [comment]
Can someone tell me what the default value is for storagePath on a fresh install (Windows)?
